collector/os_release.go

```diff
@@ -0,0 +1,175 @@
+// Copyright 2021 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collector
+
+import (
+	"errors"
+	"io"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+	envparse "github.com/hashicorp/go-envparse"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	etcOSRelease    = "/etc/os-release"
+	usrLibOSRelease = "/usr/lib/os-release"
```

Comment:
Couldn't both of these be made constants? I see nowhere in the code that modifies them.

Reply:
Good catch. They were meant as constant. Fixed.